paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -141,6 +144,12 @@ internal class PaymentSheetViewModelTest {
     @get:Rule
     val sessionRule = SessionTestRule()
 
+    @get:Rule
+    val cvcRecollectionFeatureRule = FeatureFlagTestRule(
+        featureFlag = FeatureFlags.cvcRecollection,
+        isEnabled = true
```

Comment:
Can you add tests that ensure things don't fire CVC when this feature flag is false?

Reply:
I can but the flag and related tests will be removed before the next release

Reply:
👍